modules/smart-agent_winprocess/conf/readme.yaml

```diff
@@ -0,0 +1,3 @@
+documentations:
```

Comment:
Please add links to documentation: https://github.com/signalfx/integrations/blob/master/signalfx-agent/agent_docs/monitors/telegraf-win_services.md